priv/repo/migrations/20180618202035_create_gateway_identifiers_table.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Console.Repo.Migrations.CreateGatewayIdentifiersTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:gateway_identifiers, primary_key: false) do
```

Comment:
so these won't just be for gateways. We'll want to be able to apply these identifiers to any hardware our OUI is responsible for and keep them unique across all of those things. right now that's devices and gateways but we want to keep it flexible